themes/cardinal_service/lib/scss/favorite/_dashboard.scss

```diff
@@ -0,0 +1,196 @@
+@charset 'UTF-8';
+
+.su-masthead {
+  @include grid-media-min('lg') {
+    .su-lockup {
+      flex: 0 0 50%;
+      max-width: 50%;
+    }
+
+    .su-site-search {
+      flex: 0 0 25%;
+      max-width: 25%;
+    }
+  }
+}
+
+.role--anonymous {
+  .user-links {
+    width: max-content;
+    border: 1px solid #e5e5e5;
+    border-radius: 20px;
+    padding: 6px 20px;
+    margin: 0.2em 0;
+
+    &::before {
+      font-family: 'Font Awesome 5 Free';
+      font-weight: 900;
+      content: '\f2bd';
+      color: #4d4f53;
+      margin-right: 8px;
+    }
+
+    a {
+      text-decoration: none;
+      color: #4d4f53;
+
+      &:hover,
+      &:focus {
+        text-decoration: underline;
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+.user-links {
+  width: 25%;
+  height: fit-content;
+  z-index: 101;
```

Comment:
Not sure if this is the issue, but when I open the links, i can only click on the top part of the "Log Out" link. It seems when I hover over the lower half of the log out link, the menu closes.

Reply:
This is the only thing remaining that i can tell. otherwise great job with the fixes

Reply:
This `z-index` issue should be resolved in our next core release. 😄 
[See stanford_basic PR](https://github.com/SU-SWS/stanford_basic/pull/166)